client/src/components/atoms/style/TextField.scss

```diff
@@ -7,14 +7,15 @@
 
   &-input {
     align-self: left;
-    border: 1px solid $input-border;
+    border: 1px solid $dark-border-color;
```

Comment:
confirmed with bonnie that for all textfields, their border should be $dark-border-color when unfocused on $tpc-green whn focused